app/models/metric.rb

```diff
@@ -133,14 +133,16 @@ def unit=(value)
     self[:unit] = value unless child?
   end
 
+  def type
+    method_metric? ? 'method' : 'metric'
```

Comment:
```suggestion
    @type ||= method_metric? ? 'method' : 'metric'
```

Reply:
I would also suggest using `:method` and `:metric` instead, but it probably breaks something.

Reply:
Yeah, both suggestions are good. As this method is new, I can certainly ensure, that I `.to_s` is called whenever a string is expected.

Reply:
Done.